tests/room_test.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+import rhino3dm
+from pathlib import Path
+from honeybee.room import Room
+from honeybee_3dm.room import to_room
+
+
+def test_room():
+    path = str((Path.cwd()).joinpath('tests', 'test.3dm'))
+    room = to_room(path)[0]
+    assert isinstance(room, Room)
```

Comment:
Same comments here. You should write some better tests here. This can pass even if the translation is not what you want it to be.